src/components/Company/CompanyName/CorrectNameRequest.vue

```diff
@@ -1,57 +1,90 @@
-    ></v-text-field>
-    <v-text-field
-      v-model="entityPhone"
-      filled
-      label="Enter the Applicant Phone Number"
-      hint="Example: 555-555-5555"
-      persistent-hint
-      type="tel"
-      :rules="entityPhoneNumberRules"
-      data-test="entity-phone"
-    ></v-text-field>
-    <div class="font-weight-bold ml-3 mb-2">or</div>
-    <v-text-field
-      v-model="entityEmail"
-      filled
-      label="Enter the Applicant Email Address"
-      hint="Example: name@email.com"
-      persistent-hint
-      :rules="entityEmailRules"
-      data-test="entity-email"
-    >
-    </v-text-field>
+    <v-layout row>
+      <v-flex md1 class="pa-3 pl-0">
+        <v-btn x-small fab outlined disabled color="#1A5A96" class="step-icon">1</v-btn>
+      </v-flex>
+      <v-flex>
+        <v-text-field
+          v-model="nameRequestNumber"
+          filled
+          label="Enter the NR Number"
+          hint="Example: NR 1234567"
+          req
+          persistent-hint
+          :rules="entityNumRules"
+          data-test="business-identifier"
+        ></v-text-field>
+      </v-flex>
+    </v-layout>
+    <v-layout row>
+      <v-flex md1 class="pa-3 pl-0">
+        <v-btn x-small fab outlined disabled color="#1A5A96" class="step-icon">2</v-btn>
+      </v-flex>
+      <v-flex md5>
+        <v-text-field
+          v-model="entityPhone"
+          filled
+          label="Applicant's Phone Number"
+          hint="Example: 555-555-5555"
+          persistent-hint
+          type="tel"
+          :rules="entityPhoneNumberRules"
+          data-test="entity-phone"
+        ></v-text-field>
+      </v-flex>
+      <div class="ma-5">or</div>
+      <v-flex>
+        <v-text-field
+          v-model="entityEmail"
+          filled
+          label="Applicant's Notification Email"
+          hint="Example: name@email.com"
+          persistent-hint
+          :rules="entityEmailRules"
+          data-test="entity-email"
+        >
+        </v-text-field>
+      </v-flex>
+    </v-layout>
   </v-form>
 </template>
 
 <script lang="ts">
 // Libraries
 import { Component, Prop, Watch, Emit, Mixins } from 'vue-property-decorator'
-import { Action } from 'vuex-class'
+import { Action, Getter, State } from 'vuex-class'
 
 // Mixins
 import { NameRequestMixin } from '@/mixins'
 
-// Interfaces
-import { NrCorrectionIF } from '@/interfaces/correction-interfaces'
-import { ActionBindingIF } from '@/interfaces'
+// Interfaces & Enums
+import {
+  ActionBindingIF,
+  BusinessInformationIF,
+  NameRequestApplicantIF,
+  NameRequestIF,
+  NrCorrectionIF
+} from '@/interfaces'
+import { CorrectionTypes } from '@/enums'
 
 @Component({})
 export default class CorrectNameRequest extends Mixins(NameRequestMixin) {
   /** Form Submission Prop */
-  @Prop({ default: false }) submit: boolean
+  @Prop({ default: false }) submitId: string
+
+  // Global state
+  @State(state => state.stateModel.nameRequest)
+  readonly nameRequest!: NameRequestIF
```

Comment:
We should just have a getter for this data.
Same for "businessInformation" below.

Reply:
kk, copy that, will update this!

Reply:
So much better now! 👍